### ISSUE TITLE
Using other face detectors for face attribute detection

### ISSUE THREAD

**Issue by shekarneo** [deepinsight/insightface#2348]
Hello, can I use any other face detectors for age and gender detection? Will the results be the same if I do the preprocessing the same way as here?

**Comment by Devang-C** [comment]
> 

Yes you can use other face detectors while preprocessing the images as done in this repository. Although you will have to be careful because face detectors while detecting faces take input images in a certain size, and there you might get compatibility issues. So make sure to use resize function to resize the images if needed
